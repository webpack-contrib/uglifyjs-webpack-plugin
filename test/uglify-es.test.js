import uglifyEsPackageJson from 'uglify-es/package.json';
import { dependencies } from '../package-lock.json';

describe('uglify-es', () => {
  const { version, integrity } = dependencies['uglify-es'];

  it('matches version in package-lock.json', () => {
    expect(uglifyEsPackageJson.version).toEqual(version);
  });

  it('matches integrity in package-lock.json', () => {
    // eslint-disable-next-line no-underscore-dangle
    expect(uglifyEsPackageJson._integrity).toEqual(integrity);
  });

  it('matches version snapshot', () => {
    expect(uglifyEsPackageJson.version).toMatchSnapshot();
  });

  it('matches integrity snapshot', () => {
    // eslint-disable-next-line no-underscore-dangle
    expect(uglifyEsPackageJson._integrity).toMatchSnapshot();
  });
});
