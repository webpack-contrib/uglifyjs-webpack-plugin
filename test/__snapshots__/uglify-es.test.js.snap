// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uglify-es matches integrity snapshot 1`] = `"sha512-hJ0lb3CoazL8TANWJCb6TusMOvcBSZp5vRiA3PST+LzwedUOguvf4xcpPwQGRS5GnaLYll9DQEL/jr8HPAioMg=="`;

exports[`uglify-es matches version snapshot 1`] = `"3.0.21"`;
